tests/vars.sample.yaml

```diff
@@ -1,6 +1,11 @@
 # Path to install_yamls repo
 install_yamls_path: ~/install_yamls #CUSTOMIZE_THIS
 
+src_env_bastion_key: ~/.ssh/id_src_cloud #CUSTOMIZE_THIS
```

Comment:
Should we have these two src_env_bastion variables provided as defaults, or is it ok leaving them out from here? Maybe going forward we should prefer overriding the `openstack_ssh_command` (or `source_openstack_command`) as a whole, without going through the intermediate variables... It kinda evolved organically and we need them to not break existing jobs, but i don't know if it's the best pattern to keep expanding on...